docs/windows.md

```diff
@@ -125,7 +125,10 @@ document.
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
 ```
 
-#### Add Windows kube-proxy DaemonSet
+#### Add Windows kube-proxy DaemonSet (for Kubernetes versions prior to 1.26)
+
+Windows kube-proxy will no longer be supported by Antrea starting from Kubernetes 1.26. This decision is based on the deprecation of the userspace mode,
```

Comment:
```suggestion
Windows kube-proxy is no longer supported by Antrea since version 1.13. This decision is based on the deprecation of the kube-proxy userspace mode in Kubernetes,
```

Reply:
But kuberenetes does not support kube-proxy any more, which kube-proxy userspace version would you suggest the users to use?

Reply:
After discussion, we still have a question regarding whether to maintain windows kube-proxy in document. While Antrea 1.13 sets "proxyall" as the default value for the Windows agent, it still maintains support for kube-proxy in Kubernetes versions prior to 1.26. Do you have any suggestions for updating the Windows documentation? @jianjuns @antoninbas

Reply:
First, we should definitely enable the ToC for this document, as it is getting a bit confusing to navigate :)

Then, I think that we can still mention that mode (userspace kube-proxy) for the time being, but given that proxyAll also works for older K8s versions, it's probably better to have this information later in the document. Let's move it outside of the `### Installation as a Pod (Docker/Containerd based runtimes)` section.

This section (`### Installation as a Pod (Docker/Containerd based runtimes)`) should make the following clear:

> By default, Antrea will take over all the responsibilities of kube-proxy for Windows worker Nodes, and kube-proxy should not be deployed on Windows worker Nodes to avoid conflicts with Antrea. For K8s versions prior to 1.16, Antrea can coexist on Windows Nodes with user-space kube-proxy; for more information refer to section ...

You may want to put the new section right before `### Manually run kube-proxy and antrea-agent on Windows worker Nodes`.